src/test/resources/mapping/expectedTransformationFields.json

```diff
@@ -321,7 +321,39 @@
 			"displayNameKey": "item.yearCaption",
 			"path": "$.item.yearCaption",
 			"recordType": "ITEM"
-		}
-	],
-	"totalRecords": 49
+		},
+    {
+      "fieldId": "item.callnumber",
+      "displayNameKey": "item.callNumber",
+      "path": "$.item.effectiveCallNumberComponents.callNumber",
+      "recordType": "ITEM"
+    },
+    {
+      "fieldId": "item.callnumberprefix",
+      "displayNameKey": "item.callNumberPrefix",
+      "path": "$.item.effectiveCallNumberComponents.prefix",
+      "recordType": "ITEM"
+    },
+    {
+      "fieldId": "item.callnumbersuffix",
+      "displayNameKey": "item.callNumberSuffix",
+      "path": "$.item.effectiveCallNumberComponents.suffix",
+      "recordType": "ITEM"
+    },
+    {
+      "fieldId": "item.callnumbertype.dewey.decimal.classification",
+      "displayNameKey": "item.callNumberType",
+      "referenceDataValue": "Dewey Decimal classification",
+      "path": "$.item.effectiveCallNumberComponents.typeId",
+      "recordType": "ITEM"
+    },
+    {
+      "fieldId": "item.callnumbertype.lc.modified",
```

Comment:
This data means that we are allowing for each call number type to go to a different field, Should it be that way? or is it just that we need to map the entered call number type id to value. Both are quite different. Can you confirm that once

Reply:
I think it should work like instance.identifiers, @andrii-novytskyi  what do you think ?

Reply:
Confirmed with Magda, it should not work like instance.identifiers, no need to use reference data